projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -0,0 +1,138 @@
+import { AdminSectionHero, Container, Col, Text, H3, Button } from '@automattic/jetpack-components';
+import { useProductCheckoutWorkflow } from '@automattic/jetpack-connection';
+import { Spinner, Popover } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon, help } from '@wordpress/icons';
+import classnames from 'classnames';
+import React, { useState, useCallback } from 'react';
+import useAnalyticsTracks from '../../hooks/use-analytics-tracks';
+import { JETPACK_SCAN } from '../admin-page';
+import styles from './styles.module.scss';
+
+const FirewallHeader = ( { status, hasRequiredPlan } ) => {
+	const { adminUrl } = window.jetpackProtectInitialState || {};
+
+	const { run } = useProductCheckoutWorkflow( {
+		productSlug: JETPACK_SCAN,
+		redirectUrl: adminUrl,
+	} );
+
+	const { recordEventHandler } = useAnalyticsTracks();
+	const getScan = recordEventHandler( 'jetpack_protect_waf_header_get_scan_link_click', run );
+
+	const [ showPopover, setShowPopover ] = useState( false );
+
+	const handleEnter = useCallback( () => {
+		setShowPopover( true );
+	}, [] );
+
+	const handleOut = useCallback( () => {
+		setShowPopover( false );
+	}, [] );
+
+	if ( 'on' === status ) {
+		return (
+			<AdminSectionHero>
+				<Container
+					className={ styles[ 'firewall-header' ] }
+					horizontalSpacing={ 6 }
```

Comment:
```suggestion
					horizontalSpacing={ 7 }
```
Should be `56px` per the Figma designs :+1:

<img width="218" alt="Screen Shot 2022-11-22 at 11 29 20 AM" src="https://user-images.githubusercontent.com/10933065/203393339-e4898992-4eb8-4a19-bc3a-5cf7241322cc.png">